server/api/settings.py

```diff
@@ -184,9 +184,9 @@
 # EMAIL
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = os.environ.get("EMAIL_HOST")
-EMAIL_PORT = int(os.environ.get("EMAIL_PORT"))
-EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
-EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
-EMAIL_USE_TLS = os.environ.get("APP_ENV") == "True"
 EMAIL_ADDRESS_FROM = os.environ.get("EMAIL_ADDRESS_FROM")
+EMAIL_HOST_PASSWORD = os.getenv("SENDGRID_API_KEY")
```

Comment:
There no SENDGRID_API_KEY in dev - is this still going to work?

Reply:
> There no SENDGRID_API_KEY in dev - is this still going to work?

yes, I know, I have seen it and didn't change, because it's not used anywhere so far, and I kept so less changes are made in the future, only end prod should be added

Reply:
> There no SENDGRID_API_KEY in dev - is this still going to work?

although it was working fine, I returned the old naming just to avoid confusion